packages/demo-cards/card-space-emily/card.json

```diff
@@ -1 +1,31 @@
-{"adoptsFrom":"../card-space-layout","data":{"profile":{"username":"Emily"}}}
+{
+  "adoptsFrom": "../card-space-layout",
+  "data": {
+    "background": {
+      "coverPhoto": "[COVER PHOTO]"
```

Comment:
The card space example has photo, link, and text area field types. I'm wondering if it's a good time to start creating those after this PR is merged, or if there's more work that needs to be done before we start adding more field types.

Reply:
I don't think there is any reason to wait. It sounds like you're work on the containsMany editing UI is also important, but that they're orthogonal